src/mme-app/handlers/attach_aia_ula.c

```diff
@@ -303,9 +302,10 @@ post_to_next(int ue_index)
 	}
     else if (STAGE1_AIA_FAIL == ue_entry->ue_state)
     {
-        log_msg(LOG_ERROR, "Error AIA from HSS\n");
-        log_msg(LOG_INFO, "Sending Attach Reject\n");
-	    struct s1ap_common_req_Q_msg s1ap_rej = {0};
+        log_msg(LOG_ERROR, "Error AIA from HSS. Release UE session \n");
+        release_ue_entry(ue_entry); 
```

Comment:
Yes.. Looks like when I coded this portion, rejection code was under #if 0. But since code is not enabled, my code to release the UE should be moved down.